docker/Dockerfile-carla-mot

```diff
@@ -0,0 +1,35 @@
+ARG base_image_tag
+
+FROM twosixarmory/pytorch:${base_image_tag} AS armory-bytetrack
```

Comment:
Should `armory-bytetrack` be changed to `carla-mot` here as well?  not sure if it matters

Reply:
Good catch. I'm mostly sure it doesn't matter, but fixing that will stop the next person from having to wonder.

Reply:
The `AS` clause is only needed for multi-stage builds, and was not needed here.

[Dockerfile reference](https://docs.docker.com/engine/reference/builder/#from:~:text=Optionally%20a%20name%20can%20be%20given%20to%20a%20new%20build%20stage%20by%20adding%20AS%20name%20to%20the%20FROM%20instruction.%20The%20name%20can%20be%20used%20in%20subsequent%20FROM%20and%20COPY%20%2D%2Dfrom%3D%3Cname%3E%20instructions%20to%20refer%20to%20the%20image%20built%20in%20this%20stage.)